### ISSUE TITLE
dataset

### ISSUE THREAD

**Issue by Phoebe-star** [jhb86253817/tf-re-id#1]
I download the  cuhk03 dataset   . but it  is .mat file . it  can not run 
and  do you have  trained-model  ?  how about your cmc result?

**Comment by jhb86253817** [comment]
@Phoebe-star Hi, we didn't keep the trained models since there are too many of them (10 or 20 seeds for each dataset), and we only recorded rank 1,5 and 10 of CMC results. If you'd like to train the model, please extract the images from the .mat file, then save them in the format as illustrated in the ReadMe.

**Comment by Phoebe-star** [comment]
can you provide your cuhk03 dataset ?
and how about your rank-1 for training data?

**Comment by jhb86253817** [comment]
@Phoebe-star  I'm not allowed to further distribute the data (see the description on CUHK03 download web page). I didn't check the rank-1 on training data, and it is not comparable to the test data because there are much more ID classes in the training data. If using all the IDs to evaluate rank-1, then it is a more difficult task on training data.

**Comment by Phoebe-star** [comment]
thanks 
can you tell me how to prepare data to train?

can you give me some  cuhk03 file name  example for  after preparing?
and the lable of example  person image ? ( if you have do the thing 

this is my assume
picture1   name  1_001_1_01.png     , its label is  1   in txt
picture2   name  1_007_1_05.png     , its label is  7   in txt

**Comment by jhb86253817** [comment]
Hi, the thing you need to do is to write a script to extract the images from the original .mat file, and resize them to size 128x48. Then store them to jpg files, and put them under folder 'images/cuhk03/All_128x48/detected/'. As for the image names, your assumption is correct, you can also refer to the folder 'images_fake', although the images there are fake, but the image names are the same as the real images. If you train with my code, then it's not necessary to prepare labels since my code will generate them according to the image names.

**Comment by Phoebe-star** [comment]
thanks, I will try it

**Comment by Phoebe-star** [comment]
how do you resize dataset?
you directly resize? not do crop?
and how about your dataset extend?

**Comment by jhb86253817** [comment]
Hi, I simply resize it, no cropping. As for the data augmentation, it's included in my code, you don't need to do it.

**Comment by Phoebe-star** [comment]
do you have use imagenet pretrain model?

and what do you think  use pretrain ,the weight will how to change?
if I use resnet 50 , and  exact the  (5c layer , no pooling ,) then connect my design network ,
the weight will change resnet50? or  it only change my design network ,no resnet50 ?

**Comment by jhb86253817** [comment]
Hi, I didn't use ImageNet pretrain model in this repo. Changing which part of the network depends on the code itself, and if you just train it in a regular way in TensorFlow, it will update the whole network.
